html-web-page/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+
+  <!-- your HTML code here 👇 -->
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>Here at JOTMC Inc., we are constantly striving to support our customers with the highest-quality and most unique jellies on a monthly basis. For the jelly lover in your life (or yourself!),
+    this is the gift that keeps on giving all year round. We offer three tiers of monethly jelly joy, and each one offers their own perks. Here is a breakdown of each, as well as your monthly price:
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated jellies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commermorative mason jars</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bonus jar of preserves bi-monthly</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to jelly Con</td>
+        <td></td>
+        <td></td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <form>
+    <input type="text" id="name" name="your-name" placeholder="Your Name" required>
+    <select name="plan" id="plan-option" required>
+      <option value="0">Select Your Plan:</option>
+      <option value="1">Basic Plan</option>
+      <option value="2">Deluxe Plan</option>
+      <option value="3">Jelly Giant Plan</option>
```

Comment:
This isn't wrong - but I'd suggest to make your `value` something like `basic`, `deluxe`, or `jelly` so that it is easier to distinguish when you eventually send the data to your backend. Something more human readable so you'll know what those values mean when you store it in a database for example.